File: src/main/java/kr/co/lunatalk/domain/dashboard/dto/response/DailyOrderCountResponse.java
package kr.co.lunatalk.domain.dashboard.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;

import java.time.LocalDate;

public record DailyOrderCountResponse(
	@Schema(description = "날짜")
	LocalDate date,

	@Schema(description = "주문 수")
	Long orderCount
) {
}

File: src/main/kotlin/kr/co/lunatalk/domain/dashboard/dto/response/DailyOrderCountResponse.kt
package kr.co.lunatalk.domain.dashboard.dto.response

import io.swagger.v3.oas.annotations.media.Schema
import java.time.LocalDate

data class DailyOrderCountResponse(
    @Schema(description = "날짜")
    val date: LocalDate,

    @Schema(description = "주문 수")
    val orderCount: Long,
)
